noxfile.py

```diff
@@ -14,13 +14,14 @@ def build(session, autobuild=False):
     session.env["SPHINXOPTS"] = "-W"
 
     shutil.rmtree("build", ignore_errors=True)
```

Comment:
I'm confused why that patch didn't recover the line. Let's try like this:
```suggestion
    shutil.rmtree("build", ignore_errors=True)

```